packages/flutter_tools/lib/src/base/command_help.dart

```diff
@@ -76,6 +76,13 @@ class CommandHelp {
     'debugProfileWidgetBuilds',
   );
 
+  CommandHelpOption _b;
+  CommandHelpOption get b => _b ??= _makeOption(
+    'b',
+    'Toggle the platform brightness setting.',
```

Comment:
I didn't know what "[brightness](https://api.flutter.dev/flutter/dart-ui/Brightness-class.html)" meant, maybe 'Toggle the platform brightness (dark and light mode).'?

Reply:
Good idea, fixed